centos-ci/bootstrap.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+if [[ $(id -u) != 0 ]]; then
+   echo "$0 needs to be run as root" ; exit 1
+fi
+
+pip install ansible==2.2.0
```

Comment:
I'll update the bootstrap script once I get access to the CI as there might be more work to do